python/pyspark/context.py

```diff
@@ -254,7 +253,7 @@ def _ensure_initialized(cls, instance=None, gateway=None):
     def __getnewargs__(self):
         # This method is called when attempting to pickle SparkContext, which is always an error:
         raise Exception(
-            "It appears that you are attempting to reference SparkContext from a broadcast "
```

Comment:
I think that the old text was correct, no? Did you mean to suggest "refer _to_" instead?